srv/modules/runners/mgr_orch.py

```diff
@@ -125,12 +140,33 @@ def describe_service(role=None, service_id=None, node=None):
                 # skips IDs we don't care about if one was passed in
                 continue
             # pylint: disable=fixme
-            # TODO: consider making this be a list if it's possible for DeepSea
+            # TODO: consider wrapping this in a list if it's possible for DeepSea
             # to deploy multiple services of the same type but with different IDs
             # on the same node.  Of course, DeepSea will deploy mutiple OSDs on
-            # the same node, but describe_service doesn't support quering OSDs,
+            # the same node, but describe_service doesn't support querying OSDs,
             # so that example doesn't count ;-)
-            minion_result[minion_role] = daemon_ids[minion]['host']
+            minion_result[minion_role] = {
+                'service_instance': daemon_ids[minion]['host']
+            }
+            if minion_role == 'ganesha':
+                if not nfs_pool:
+                    nfs_pool = _run_master_module_function('master.find_pool', ['cephfs', 'rgw'])
+                minion_result[minion_role]['rados_config_location'] = (
+                    'rados://{}/ganesha/conf-{}'.format(nfs_pool, daemon_ids[minion]['host']))
+            if minion_role == 'igw':
+                igw_address = local.cmd(minion, 'public.address', [])[minion]
+                igw_username = local.cmd(minion, 'pillar.get',
+                                         ['ceph_iscsi_username'])[minion] or 'admin'
+                igw_password = local.cmd(minion, 'pillar.get',
+                                         ['ceph_iscsi_password'])[minion] or 'admin'
+                igw_port = local.cmd(minion, 'pillar.get',
+                                     ['ceph_iscsi_port'])[minion] or '5000'
+                igw_ssl = local.cmd(minion,
+                                    'pillar.get', ['ceph_iscsi_ssl'])[minion] or False
+                protocol = 'https' if igw_ssl else 'http'
+                minion_result[minion_role]['service_url'] = (
+                    '{}://{}:{}@{}:{}'.format(protocol, igw_username, igw_password,
```

Comment:
If `igw_address` is an IPv6 address it must be enclosed in square brackets in the URL

Reply:
Thanks, will fix.